modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/version/DefaultVersionAPITestCase.java

```diff
@@ -77,95 +73,78 @@ public DefaultVersionAPITestCase(TestUserMode userMode) {
-        apiRequest.setDefault_version_checked("default_version");
+        apiRequest.setDefault_version_checked("true");
         apiRequest.setVersion(apiVersion);
         apiRequest.setTiersCollection(APIMIntegrationConstants.API_TIER.UNLIMITED);
         apiRequest.setTier(APIMIntegrationConstants.API_TIER.UNLIMITED);
-        apiRequest.setProvider(provider);
-
-        //Add the API using the API publisher.
-        HttpResponse response = apiPublisher.addAPI(apiRequest);
-
-        APILifeCycleStateRequest updateRequest =
-                new APILifeCycleStateRequest(apiName, user.getUserName(),
-                                             APILifeCycleState.PUBLISHED);
-        //Publish the API
-        response = apiPublisher.changeAPILifeCycleStatus(updateRequest);
-
-        //Login to the API Store
-        apiStore.login(user.getUserName(), user.getPassword());
 
         //Add an Application in the Store.
-        response = apiStore
-                .addApplication("DefaultVersionAPP", APIMIntegrationConstants.APPLICATION_TIER.UNLIMITED, "", "");
-        verifyResponse(response);
-
-        //Subscribe the API to the DefaultApplication
-        SubscriptionRequest subscriptionRequest = new SubscriptionRequest(apiName, apiVersion, provider,
-                "DefaultVersionAPP", APIMIntegrationConstants.API_TIER.UNLIMITED);
-        response = apiStore.subscribe(subscriptionRequest);
+        HttpResponse applicationResponse = restAPIStore
+                .createApplication("DefaultVersionAPP", "Default version testing application", APIMIntegrationConstants.APPLICATION_TIER.UNLIMITED, ApplicationDTO.TokenTypeEnum.OAUTH);
 
-        //Generate production token and invoke with that
-        APPKeyRequestGenerator generateAppKeyRequest = new APPKeyRequestGenerator("DefaultVersionAPP");
-        String responseString = apiStore.generateApplicationKey(generateAppKeyRequest).getData();
-        JSONObject jsonResponse = new JSONObject(responseString);
+        applicationID = applicationResponse.getData();
 
-        //Get the accessToken which was generated.
-        String accessToken = jsonResponse.getJSONObject("data").getJSONObject("key").getString("accessToken");
+        //Create api and subscribe the API to the DefaultApplication
+        apiId = createPublishAndSubscribeToAPIUsingRest(apiRequest, restAPIPublisher, restAPIStore, applicationID,
+                APIMIntegrationConstants.API_TIER.UNLIMITED, SubscriptionDTO.StatusEnum.UNBLOCKED,
+                SubscriptionDTO.TypeEnum.API);
 
-        String  apiInvocationUrl = getAPIInvocationURLHttp(apiContext);
+        //Generate production token and invoke with that
+        ArrayList grantTypes = new ArrayList();
+        grantTypes.add("client_credentials");
+        ApplicationKeyDTO applicationKeyDTO = restAPIStore.generateKeys(applicationID, "3600", null, ApplicationKeyGenerateRequestDTO.KeyTypeEnum.PRODUCTION, null, grantTypes);
```

Comment:
Shall we move it to separate lines, seems like the length is exceeded. Please check in other places too.